default/AI/ColonisationAI.py

```diff
@@ -128,12 +140,9 @@ def check_supply():
     annexable_planet_ids.clear()
     systems_by_supply_tier.clear()
     system_supply.clear()
-    supply_distance = 0
-    for tech in AIDependencies.supply_range_techs:
-        if tech_is_complete(tech):
-            supply_distance += AIDependencies.supply_range_techs[tech]
+    supply_distance = get_supply_tech_range()
     foAI.foAIstate.misc['supply_tech'] = supply_distance
```

Comment:
At quick look `foAI.foAIstate.misc['supply_tech']` is not used and can be removed.